dnf-behave-tests/fixtures/specs/miscellaneous/non_utf_filenames-1.0-1.spec

```diff
@@ -0,0 +1,27 @@
+Name:           non_utf_filenames
+Epoch:          0
+Version:        1.0
+Release:        1
+
+License:        Public Domain
+URL:            None
+
+Summary:        A package with non-utf8 filenames
+
+BuildArch:      noarch
+
+%description
+Dummy.
+
+%install
+mkdir -p %{buildroot}/non-utf8-files/
+pushd %{buildroot}/non-utf8-files/
+  touch hex-80-�-
+  touch hex-a0-�-
```

Comment:
Might just be me but how does this create filenames that are invalid utf-8? Maybe add a comment?

Reply:
It's not visible on the github preview. There is `0x80` byte in the first file (where github shows an euro sign) and `0xa0` byte on the second line.

Reply:
Can you put it in a comment then? Even if the actual hex value is a part of the name just before the invalid bad byte, it's just not really visible in a text editor (depending on how it even displays the invalid bytes). Just clarify what's going on there...

Reply:
Done.